src/main/java/com/shippo/ExampleOrder.java

```diff
@@ -0,0 +1,86 @@
+package com.shippo;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.time.LocalDateTime;
+
+import com.shippo.exception.ShippoException;
+import com.shippo.model.Order;
+
+public class ExampleOrder {
+
+	public static void main(String[] args) throws ShippoException {
+		
+		// replace with your Shippo Token
+		// don't have one? get more info here (https://goshippo.com/docs/#overview)
+		Shippo.setApiKey("shippo_test_cf1b6d0655e59fc6316880580765066038ef20d8");
```

Comment:
do we include these in other SDKs or do we normally just do a placeholder like <API_KEY_HERE>?